lib/dfu.js

```diff
@@ -45,7 +45,6 @@ var that = module.exports = {
 	deviceID: undefined,
 	findCompatibleDFU: function () {
 		var temp = when.defer();
-		var that = this;
```

Comment:
I removed this duplicate `that` since it's already defined a couple lines above. The local `that` shadows the top-level `that`.

Ah, Javascript... https://twitter.com/bhalp1/status/578925947245633536